stores/ini/store.go

```diff
@@ -0,0 +1,321 @@
+package ini //import "go.mozilla.org/sops/stores/ini"
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/stores"
+	"gopkg.in/ini.v1"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+// Store handles storage of ini data.
+type Store struct {
```

Comment:
Is this file `goimports`/`gofmt`ed? I feel like they'd move the braces to the same line.

Reply:
Good, point. I was using an editor that doesn't automatically do this. I've run gofmt/goimports on the new files now.